client/src/common/components/GridList.jsx

```diff
@@ -25,9 +25,7 @@ class GridList extends PureComponent {
     const { addToFavourites, removeFromFavourites, route } = this.props;
 
     if (route === GridListRoutes.LIST) {
-      isFavourite
-        ? removeFromFavourites(itemId, isFavourite)
-        : addToFavourites(itemId, isFavourite);
+      isFavourite ? removeFromFavourites(itemId) : addToFavourites(itemId);
```

Comment:
I've also used ternary operator in cohorts case but I wonder it this right approach. Shouldn't ternary operator be used to return a value and not to invoke a function?